aws/data_source_aws_eip.go

```diff
@@ -149,6 +157,18 @@ func dataSourceAwsEipRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("public_ipv4_pool", eip.PublicIpv4Pool)
 
+	d.Set("public_ipv4_pool", eip.PublicIpv4Pool)
+	if eip.CustomerOwnedIpv4Pool != nil {
```

Comment:
No need for `nil` check here, `d.Set("customer_owned_ipv4_pool", eip.CustomerOwnedIpv4Pool)` is sufficient.